word_nerd/users/tests/test_register.py

```diff
@@ -3,45 +3,36 @@
 from rest_framework import status
 from rest_framework.test import APITestCase
 
+from .utils_test import register_user
+
 
 class RegisterTest(APITestCase):
-    url = reverse('register')
 
     def test_register_user(self):
-        data = {'username': 'testuser1',
-                'password': 'testnogeslo1', 'password2': 'testnogeslo1'}
-        response = self.client.post(self.url, data, format='json')
-
+        response = register_user(self.client)
         self.assertEqual(response.status_code, status.HTTP_201_CREATED)
         self.assertEqual(User.objects.count(), 1)
         self.assertEqual(User.objects.get().username, 'testuser1')
-        self.assertNotEqual(User.objects.get().password, 'testnogeslo1')
+        self.assertNotEqual(User.objects.get().password, 'coolpassyoww')
```

Comment:
Maybe check that `coolpassyoww` is not the same as the one you use (by default) in `utils_test.py`